src/number_theory/modular_forms/slash_actions.lean

```diff
@@ -51,10 +49,10 @@ namespace modular_forms
 noncomputable theory
 
 /--The weight `k` action of `GL(2, ℝ)⁺` on functions `f : ℍ → ℂ`. -/
-def slash : ℤ → GL(2, ℝ)⁺ → (ℍ → ℂ) → (ℍ → ℂ) := λ k γ f,
-  (λ (x : ℍ), f (γ • x) * (((↑ₘ γ).det) : ℝ)^(k-1) * (upper_half_plane.denom γ x)^(-k))
+def slash (k : ℤ) (γ : GL(2, ℝ)⁺) (f : ℍ → ℂ) (x : ℍ) : ℂ :=
+f (γ • x) * (((↑ₘ γ).det) : ℝ)^(k-1) * (upper_half_plane.denom γ x)^(-k)
 
-variables {Γ : subgroup SL(2,ℤ)} {k: ℤ} (f : (ℍ → ℂ))
+variables {Γ : subgroup SL(2,ℤ)} {k: ℤ} (f : ℍ → ℂ)
```

Comment:
You are not consistent with whether you put a space between `2,` and the field, I think a space is nicer.